generator/template.go

```diff
@@ -187,26 +187,26 @@ export function b64Decode(s: string): Uint8Array {
         j = 1;
         break;
       case 1:
-        buffer[offset++] = t << 2 | (c & 48) >> 4;
+        buffer[offset++] = (t << 2) | ((c & 48) >> 4);
```

Comment:
This is fixing a warning about unclear operator precedence:

    Unexpected mix of '|' and '>>'. Use parentheses to clarify the intended order of operations